src/java.base/share/classes/java/math/MutableBigInteger.java

```diff
@@ -699,14 +707,29 @@ private int mulsubBorrow(int[] q, int[] a, int x, int len, int offset) {
      * Assumes that intLen > 0, n > 0 for speed
      */
     private final void primitiveRightShift(int n) {
```

Comment:
Plz remove `final` from `private` primitive shift methods. (The `final`s were there long before your PR.)